zulipterminal/ui_tools/views.py

```diff
@@ -1939,14 +1939,21 @@ def __init__(
 
         # Get rendered message header and footer
         msg_box = MessageBox(message, controller.model, None)
+        msg_box.footer = [
+            (
+                urwid.Text(
+                    ("msg_bold", "\nPress [c] to copy message to clipboard"),
```

Comment:
I'm not sure of the wording here, but including `content` in the text seems useful, particularly if we keep the footer and header as they are in `main`.

Reply:
How's "Press [c] to copy raw message content to clipboard" ?